### PR TITLE
Finished relicense

### PR DESCRIPTION
**Pull request by TheButlah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,18 @@ It's recommended to download installer from here: https://github.com/SlimeVR/Sli
 Latest instructions are [on our site](https://docs.slimevr.dev/server-setup/slimevr-setup.html).
 
 ## License Clarification
-**SlimeVR software** (including server, firmware, drivers, installator, documents, and others - see
-licence for each case specifically) **is distributed under the MIT License ([LICENSE-MIT])
-and in some cases dually licensed under Apache-2.0 License ([LICENSE-APACHE]). These
-licenses are the copyright of Eiren Rain and the SlimeVR contributors.** 
+**SlimeVR software** (including server, firmware, drivers, installer, documents, and others - see
+licence for each case specifically) **is distributed under a dual MIT/Apache 2.0 License
+([LICENSE-MIT] and [LICENSE-APACHE]). These licenses are the copyright of the SlimeVR
```
Comment (Eirenliel) [inline README.md:22]: This is worded wrong.

```suggestion
([LICENSE-MIT] and [LICENSE-APACHE]). The software is the copyright of the SlimeVR
```

Comment (Eirenliel) [inline README.md:22]: This is worded wrong.

```suggestion
([LICENSE-MIT] and [LICENSE-APACHE]). The software is the copyright of the SlimeVR
```

**Review by Eirenliel — CHANGES_REQUESTED** [review]

**Comment by Eirenliel** [comment]
@TheButlah 🥺

**Comment by TheButlah** [comment]
oops 
will fix after work
you can also edit the PR directly

File: LICENSE-APACHE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Eiren Rain and SlimeVR Contributors
+   Copyright 2020 SlimeVR Contributors
```
Comment (TheButlah) [inline LICENSE-APACHE:189]:
```suggestion
   Copyright 2022 Eiren Rain and SlimeVR Contributors
```

File: LICENSE-MIT [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Eiren Rain and SlimeVR Contributors
+Copyright (c) 2020 SlimeVR Contributors
```
Comment (TheButlah) [inline LICENSE-MIT:3]:
```suggestion
Copyright (c) 2020 Eiren Rain and SlimeVR Contributors
```

File: README.md [hunk 4]
```diff
@@ -17,18 +17,18 @@ It's recommended to download installer from here: https://github.com/SlimeVR/Sli
...
-**However, these Licenses have some limits, and if you wish to distribute software based
+**However, these licenses have some limits, and if you wish to distribute software based
 on SlimeVR, you need to be aware of them:**
 
 * When distributing any software that uses or is based on SlimeVR, you have to provide
   to the end-user at least one of the original, unmodified [LICENSE-MIT] or
-  [LICENSE-APACHE] files from SlimeVR. This includes the `Copyright (c) 2022 Eiren Rain
-  and SlimeVR Contributors` part of the license. It is insufficient to use a generic MIT
+  [LICENSE-APACHE] files from SlimeVR. This includes the `Copyright (c) 2022 SlimeVR
+  Contributors` part of the license. It is insufficient to use a generic MIT
```
Comment (TheButlah) [inline README.md:31]:
```suggestion
  [LICENSE-APACHE] files from SlimeVR. This includes the `Copyright (c) 2020 Eiren Rain
  and SlimeVR Contributors` part of the license. It is insufficient to use a generic MIT
```

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**